apiml-security-common/src/main/java/org/zowe/apiml/security/common/filter/CategorizeCertsFilter.java

```diff
@@ -94,7 +94,7 @@ private Optional<Certificate> getClientCertFromHeader(HttpServletRequest request
                     .generateCertificate(new ByteArrayInputStream(Base64.getDecoder().decode(certFromHeader)));
                 return Optional.of(certificate);
             } catch (Exception e) {
-                apimlLog.log("org.zowe.apiml.security.common.filter.errorParsingCertificate", e.getMessage(), certFromHeader);
+                apimlLog.log("org.zowe.apiml.security.common.filter.errorParsingCertificate", request.getHeader("host"), e.getMessage(), certFromHeader);
```

Comment:
Isn't the "host" header the server header, the receiving gw?

Reply:
yes, I modified the log message.

Reply:
I don't see, how this will help to explain the situation. The host or the request url identifies the receiver that is logging the message. Why to log url of the service that is printing the message?

Reply:
here I am just giving the additional details around the Error i.e  RequestURL and changed the message from central gateway to gateway.

Reply:
Please add a request.remoteHost() to identify the client gateway.